tools/structure/src/main.py

```diff
@@ -253,7 +253,7 @@ def run(
             structured_output_dict[SettingsKeys.METADATA] = metadata
             structured_output = json.dumps(structured_output_dict)
 
-        metrics = structured_output_dict[SettingsKeys.METRICS]
+        metrics = structured_output_dict.get(SettingsKeys.METRICS, {})
```

Comment:
@Deepak-Kesavan since we always set `capture_metrics` to `True` -> why do you think this metrics key is missing?

Reply:
It was not handled for table and record type